playkit/src/main/java/com/kaltura/playkit/LocalAssetsManager.java

```diff
@@ -25,10 +25,12 @@
 public class LocalAssetsManager {
 
     private static final PKLog log = PKLog.get("LocalAssetsManager");
+    private static final String ASSET_ID_PREFIX = "assetId:";
```

Comment:
final name will be "assetId:123XYZ_type:" and the value of the key will be the type.